.gitignore

```diff
@@ -102,4 +102,10 @@ _*
 /native/**/*.vcxproj
 /native/**/*.vcxproj.filters
 /native/**/*.make
-/native/**/Makefile
\ No newline at end of file
+/native/**/Makefile
+
+# Compiled Shaders
+/native/monogame/directx12/*.mgfxo
+/native/monogame/directx12/*.mgfxo.h
+/native/monogame/vulkan/*.mgfxo
+/native/monogame/vulkan/*.mgfxo.h
```

Comment:
These still need to be committed to the repo, execpt the build machine will be doing it, don't add them to gitignore.